ArkEcosystem/laravel-fortify
# refactor: auth forms design

## Summary
Related to https://app.clickup.com/t/pbgftv

This PR makes "auth forms" in line with the current design and in particular it takes care of:
- **Title** font-size to be 36px and 30px (< 768px)
- **Subtitle** font-size to be 18px
- "**Remember me**" text to be 16px

related to this discussion https://cryptoarkproject.slack.com/archives/CLA4SFYE7/p1626853474316800:
- remove form border on mobile (375px) for: 
   - login form
   - registration form
   - reset password form (request form / reset form)
   - two factor authentication form (OTP form / recovery code form)

## How to test
1. merge this PR on [MSQ PR#2179](https://github.com/ArkEcosystem/marketsquare.io/pull/2179) and/or on [NODEM PR#111](https://github.com/ArkEcosystem/nodem/pull/111) (only "login" form)
2. run `yarn run prod`
3. refer to the lists above to check forms and the other changes

## Checklist
